partcad/tests/unit/test_git_clone_retries.py

```
@@ -84,55 +86,51 @@
          pytest.raises(RuntimeError):
 
-        ctx = pc.Context()
+        user_config = UserConfig()
+        user_config.internal_state_dir = temp_dir
+        user_config.set('git.clone.retry.max', test_git_retry_config["max"])
+        user_config.set('git.clone.retry.patience', test_git_retry_config["patience"])
+
+        ctx = pc.Context(user_config=user_config)
         factory = pc.ProjectFactoryGit(ctx, None, test_config_import_git)
-        factory._clone_or_update_repo(repo_url)
+        assert factory is not None
```

Ruff (0.8.2)

`pytest.raises()` block should contain a single simple statement

(PT012)